lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -595,16 +707,17 @@
                         // We have no other information from the platform on the cause of the exception.
                         platform.raiseOptionValueOutOfRange();
                     }
-                    if (num === 0) caseFirst = 'false';
-                    else if (num === -1) caseFirst = 'upper';
-                    else caseFirst = 'lower';
+
+                    caseFirst = (num === 0) ? 'false' : (num === -1) ? 'upper' : 'lower';
```

Comment:
I personally think `if-else` is more readable.

Reply:
Yeah I think you're right. I wanted to add braces but initially didn't like how long it got. I'll change it to the long form with braces.